libcontainerd/remote/client.go

```diff
@@ -534,6 +551,14 @@ func (c *container) createIO(fifos *cio.FIFOSet, stdinCloseSync chan containerd.
 	return rio, err
 }
 
+func closeStdin(ctx context.Context, p containerd.Process) error {
+	err := p.CloseIO(ctx, containerd.WithStdinCloser)
+	if err != nil && strings.Contains(err.Error(), "transport is closing") {
```

Comment:
Do we know if this returned a typed error for this case (i.e., would there be an option to avoid string-matching here?);
https://github.com/moby/moby/blob/816cd9802fba47611a3e4ec2c25409b497a4daeb/vendor/github.com/containerd/containerd/task.go#L406-L408